File: Selenium/Activities/Activity_8.py
import time

from selenium import webdriver
from selenium.webdriver.common.by import By

with webdriver.Chrome() as driver:
    driver.get("https://v1.training-support.net/selenium/dynamic-controls")
    print("The page title is: ", driver.title)
    driver.find_element(By.ID, "toggleCheckbox").click()
    driver.find_element(By.ID, "toggleCheckbox").click()
    driver.find_element(By.XPATH, "//input[@class='willDisappear']").click()
    time.sleep(5)
    driver.quit()

File: Selenium/Activities/Activity_8.java
package Activities;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.chrome.ChromeDriver;

import java.time.Duration;

public class Activity_8 {
    public static void main(String[] args) {
        WebDriver driver = new ChromeDriver();
        driver.get("https://v1.training-support.net/selenium/dynamic-controls");
        System.out.println("The title of page is: " + driver.getTitle());
        driver.findElement(By.id("toggleCheckbox")).click();
        driver.findElement(By.id("toggleCheckbox")).click();
        driver.findElement(By.xpath("//input[@class='willDisappear']")).click();
        driver.quit();


    }
}
